sphinxcontrib/openapi/openapi30.py

```diff
@@ -246,6 +248,49 @@ def _httpresource(endpoint, method, properties, convert, render_examples,
             yield '{indent}{line}'.format(**locals())
         yield ''
 
+    securities = properties.get('security', [])
+    for security in securities:
+        yield ('{indent}This resource requires the '
+               'following authentication scheme(s):').format(**locals())
+        yield ''
+        for ref in security.keys():
+            secScheme = components.get('securitySchemes', {}).get(ref, {})
+            if secScheme['type'] == 'http':
+                if secScheme['scheme'] == 'basic':
```

Comment:
If I'm not mistaken, `secScheme['scheme']` is case insensitive. Can you please`.lower()` it before comparison? Otherwise, it won't work for specs with `Basic` spelling and that's something I'd like to address.